ethers-signers/src/trezor/app.rs

```diff
@@ -56,9 +64,39 @@ impl TrezorEthereum {
         Ok(())
     }
 
+    fn get_cached_session() -> Result<Option<Vec<u8>>, TrezorError> {
+        let mut session = [0; SESSION_ID_LENGTH];
+
+        let path = env::current_dir()
+            .map_err(|e| TrezorError::CacheError(e.to_string()))?
+            .join("cache")
+            .join("trezor.session");
```

Comment:
Maybe we should pass a `cache_dir` to trezor when initializing it? Or have a global `~/.ethers-rs/trezor/cache/trezor.session` file?

Reply:
added all options
1. accepts `cache_dir` as a parameter
2. if none, then tries `home_dir`
3. if none, then tries `current_dir`
4. if err, exit